### ISSUE TITLE
pull to refresh not working with custom view that has a height of 200

### ISSUE THREAD

**Issue by echoedexistence** [samvermette/SVPullToRefresh#103]
For some reason the SVPullToRefresh height is predetermined by a constant

**Comment by samvermette** [comment]
Indeed, I never planned for the view to have other dimensions. Feel free to submit a pull request that makes it work with other heights (we should probably be able to set its `frame` property).

**Comment by yimingtang** [comment]
By the way, I think we'd better support rotation for custom view.
